pkg/operator/ceph/nfs/config.go

```diff
@@ -117,19 +117,24 @@ NFSv4 {
 }
 
 RADOS_KV {
-	ceph_conf = '` + cephclient.DefaultConfigFilePath() + `';
+	ceph_conf = "` + cephclient.DefaultConfigFilePath() + `";
 	userid = ` + userID + `;
 	nodeid = ` + nodeID + `;
 	pool = "` + n.Spec.RADOS.Pool + `";
 	namespace = "` + n.Spec.RADOS.Namespace + `";
 }
 
 RADOS_URLS {
-	ceph_conf = '` + cephclient.DefaultConfigFilePath() + `';
+	ceph_conf = "` + cephclient.DefaultConfigFilePath() + `";
 	userid = ` + userID + `;
 	watch_url = '` + url + `';
 }
 
+RGW {
+        name = "client.` + userID + `";
+        cluster = "ceph";
```

Comment:
Did you try using the `ceph_conf` config here instead of `cluster`? I think `ceph_conf` would allow us to update the connection in the future if necessary and has important info like the keyring.

Reply:
for ceph configuration is set as default string in other sections, ceph_conf seems not necessary for current implementation. Maybe in future it could be iterated further, for now should be enough for simple working configuration.

Reply:
@BlaineEXE any other concerns?

Reply:
It's important make sure the RGW config doesn't have unnecessary configs that would make user-defined modifications hard or impossible to apply. Additionally, we do need necessary configs. If things are working without specifying `ceph_conf`, I think we can go ahead without specifying it. 

What I would ask in follow-up is why `cluster = "ceph";` is necessary since that is the default value. This should be non-necessary, so let's remove it unless there is a specific need.

Reply:
removed 'cluster = "ceph"' for cleanness